### ISSUE TITLE
Pyam dependency

### ISSUE THREAD

**Issue by rgieseke** [openclimatedata/openscm#161]
When doing a clean install from Master i get a message about Pyam:

```
ERROR: pyam-iamc 0.1.2+39.ga6ac0c5 has requirement pandas<=0.23.4,>=0.21.0, but you'll have pandas 0.24.2 which is incompatible.
```

**Comment by znicholls** [comment]
should be fixed thanks to pyam v0.2.0 so can close?

**Comment by rgieseke** [comment]
Yup, thanks for the follow-up!
